unattended-upgrade

```diff
@@ -1535,8 +1598,28 @@ def main(options, rootdir=""):
     else:
         logging.debug("dpkg is configured not to cause conffile prompts")
 
-    # auto-removal
-    previous_autoremovals = get_auto_removable(cache)
+    # get log
+    logfile_dpkg = os.path.join(_get_logdir(), 'unattended-upgrades-dpkg.log')
+    if not os.path.exists(logfile_dpkg):
+        with open(logfile_dpkg, 'w'):
+            pass
+
+    # auto-removals
+    auto_removable = get_auto_removable(cache)
+    if (auto_removable and apt_pkg.config.find_b(
+            "Unattended-Upgrade::Remove-Unused-Kernel-Packages", True)):
```

Comment:
I'm curious, why `True` here but its overriden in all the configs (in the data dir) to `false` ?

Reply:
This seems to be the pattern for options enabled by default. ```if apt_pkg.config.find_b(..., True)``` in the code and ```//... "false"``` in the configs.

Reply:
Indeed it is, sorry. These are the commented out options.